package.json

```diff
@@ -26,7 +26,7 @@
     "compile": "node ./node_modules/vscode/bin/compile -watch -p ./"
   },
   "devDependencies": {
-    "typescript": "^3.1.6",
+    "typescript": "^3.2.1",
```

Comment:
This seems like a very specific version to depend on. Is there some particular feature we need in that version?  (I know nothing about the VS Code / Typescript ecosystem, so I'm just asking...

Reply:
No, there is not a good rationale for this. I will attempt to find one.

Reply:
@housel has the solution [here](https://gist.github.com/pedro-w/9558d1d5de14ef7dda247c588fd088f8).

The current minimum version of TS supported by the extension now is 2.0.3. So I will update `package.json` accordingly.